voltdb/src/main/java/org/voltdb/sortedvolttable/ClientResponseIsBadException.java

```diff
@@ -0,0 +1,29 @@
+package org.voltdb.sortedvolttable;
```

Comment:
looks like there are still some `org.voltdb` packages here instead of `com.yahoo.ycsb.db.voltdb`

Reply:
Fixed in 0ebc1ec